### ISSUE TITLE
Non-Qt/Console Compatibility?

### ISSUE THREAD

**Issue by joeleong** [joxeankoret/diaphora#62]
```
Executing function 'main'...
diaphora.py: DLL load failed: Module not found.
Traceback (most recent call last):
  File "C:\Program Files (x86)\IDA 6.95\python\ida_idaapi.py", line 509, in IDAPython_ExecScript
    execfile(script, g)
  File "diaphora.py", line 51, in <module>
    from PyQt5 import QtCore, QtGui, QtWidgets
ImportError: DLL load failed: Module not found.
```

Fails if I try to use idaw.exe instead of idaq.exe. Probably because it doesnt have Qt :)
When trying to batch export, might be nice to be able to use IDA's console ui (idaw.exe, idal etc).

**Comment by joxeankoret** [comment]
For console export, use the following environment variables:

- DIAPHORA_AUTO_DIFF: Set to 1.
- DIAPHORA_EXPORT_FILE: Where to put the exported data.
- DIAPHORA_AUTO: Set to 1.

Otherwise, it considers it's running from an interactive IDA.

**Comment by joeleong** [comment]
Great thanks! I didn't realize DIAPHORA_AUTO_DIFF could also be used for exporting.

Only thing is, when I used the above settings the script was missing the `idaapi` functions such as `hide_wait_box`. The following hack/patch seemed to work though:

```
diff --git a/diaphora.py b/diaphora.py
index 9eb69d4..9e35646 100644
--- a/diaphora.py
+++ b/diaphora.py
@@ -3914,6 +3914,9 @@ if __name__ == "__main__":
     if file_out is None:
       raise Exception("No export file specified!")
 
+    if os.getenv("DIAPHORA_AUTO_DIFF") is not None:
+      from idaapi import *
+
     use_decompiler = os.getenv("DIAPHORA_USE_DECOMPILER")
     if use_decompiler is None:
       use_decompiler = False
```

**Comment by joxeankoret** [comment]
Thanks for the patch! I will be on holidays until next week. I will upload a fix by next week.

**Comment by joxeankoret** [comment]
@joeleong, with the previously merged patch, that bug is out, right?

**Comment by joeleong** [comment]
Yes, should be fixed with a31fbb9cc90f21dc797dfa0aa0c93ad5e8594c4b

Thanks!
